core/src/main/kotlin/behavior/interaction/response/MessageInteractionResponseBehavior.kt

```diff
@@ -22,6 +22,14 @@ import kotlin.contracts.contract
  */
 public interface MessageInteractionResponseBehavior : FollowupPermittingInteractionResponseBehavior {
 
+    /**
+     * Requests to delete the message.
+     * @throws RestRequestException if something went wrong during the request.
+     */
```

Comment:
```suggestion
    /**
     * Requests to delete the message.
     *
     * This [MessageInteractionResponseBehavior] can still be used to send followup messages to the interaction.
     *
     * @throws RestRequestException if something went wrong during the request.
     */
```
Or was there a reason that you left this part out?

Reply:
I didn't understand this "This [MessageInteractionResponseBehavior] can still..." part being documented in this function, so I removed it, but I can put it back in

Reply:
It means that after you run code like this:
```kotlin
val response: MessageInteractionResponseBehavior = interaction.respondPublic { /* ... */ }
response.delete()
```
you can still do this:
```kotlin
response.createPublicFollowup { /* ... */ }
```

If you can think of a way to better describe this, feel free to add it.

Reply:
aaah, makes sense